### PR TITLE
Update store description

### PR DESCRIPTION
**Pull request by sdrapha**
Include the fact that the app is clean and opensource, and a link, on the description.

### CODE REVIEW

File: metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -16,3 +16,6 @@ Highlight of some of the features:
 
 Like any other virtual keyboards, it must be enabled in the system settings. Open the System Settings and go to:
 System > Languages & input > On-screen keyboard > Manage on-screen keyboards.
+
+There are no ads, no tracking, no unnecessary permisions!
```
Comment (sdrapha) [inline metadata/android/en-US/full_description.txt:20]:
```suggestion
There are no ads, no tracking, no unnecessary permissions!
```

File: metadata/android/en-US/full_description.txt [hunk 2]
```diff
@@ -16,3 +16,6 @@ Highlight of some of the features:
 
 Like any other virtual keyboards, it must be enabled in the system settings. Open the System Settings and go to:
 System > Languages & input > On-screen keyboard > Manage on-screen keyboards.
+
+There are no ads, no tracking, no unnecessary permissions!
```
Comment (Julow) [inline metadata/android/en-US/full_description.txt:20]: Ads and tracking are mentioned on line 13, you might want to remove the first occurrence.
There's a lot of '!' ! I would prefer if one of them is removed.

**Comment by Julow** [comment]
F-Droid already shows a link to the repository and apps are already expected to be clean of ads and tracking. Users don't expect to see this mentioned in the description.

This only applies to the play store which is updated manually for now, I plan to try [fastlane](https://docs.fastlane.tools/actions/supply/) at some point.
In the meantime, I'd prefer if this new bit of description was in a separate file, `google_extra_description`, so it is not lost until I have a better workflow and so that F-Droid is unaffected.

File: metadata/android/en-US/full_description.txt (hunk 2 above)
Comment (sdrapha) [reply metadata/android/en-US/full_description.txt]: oh, my bad, didn't notice at the end of the line. 
I removed it.

**Comment by sdrapha** [comment]
> F-Droid already shows a link to the repository and apps are already expected to be clean of ads and tracking. Users don't expect to see this mentioned in the description.
> 
> This only applies to the play store which is updated manually for now, I plan to try [fastlane](https://docs.fastlane.tools/actions/supply/) at some point. In the meantime, I'd prefer if this new bit of description was in a separate file, `google_extra_description`, so it is not lost until I have a better workflow and so that F-Droid is unaffected.

Ah ok. I got it. So the description.txt is for fdroid only, and not play store. So yeah, these changes should have a different aproach then 👍

**Comment by Julow** [comment]
Sorry, I'm closing this PR. I'm still interested in improving the store description but it's quite low in my TODO list for this project.

**Pull request closed without merging**